Casks/g/ghostty.rb

```diff
@@ -0,0 +1,33 @@
+cask "ghostty" do
+  version "1.0.0"
+  sha256 "091f7a2b3f4160a16d7d52b2822124bb9d5714993815f62a7d70027984372652"
+
+  url "https://release.files.ghostty.org/#{version}/Ghostty.dmg"
+  name "Ghostty"
+  desc "Fast, native, feature-rich terminal emulator pushing modern features"
```

Comment:
```suggestion
  desc "Terminal emulator that uses platform-native UI and GPU acceleration"
```
See https://docs.brew.sh/Cask-Cookbook#stanza-desc.